### PR TITLE
Add the request to the Problem post-process method

### PR DESCRIPTION
**Pull request by cbornet**
This is useful to add elements from the request (eg. the request URI as is done in regular Spring Boot responses)

### CODE REVIEW

File: src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java [hunk 1]
```diff
@@ -278,4 +278,10 @@ default void log(
         return entity;
     }
 
+    default ResponseEntity<Problem> process(
```
Comment (whiskeysierra) [inline src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java]: Can you move this one before the original `process` method?

Comment (cbornet) [reply src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java]: Sure. Done.

**Comment by whiskeysierra** [comment]
:+1:

**Comment by whiskeysierra** [comment]
@cbornet Thanks for the contribution.

**Comment by whiskeysierra** [comment]
@cbornet I released your change as 0.21.0 to central.

**Pull request merged**